charts/aws-vpc-cni/templates/clusterrole.yaml

```diff
@@ -38,3 +38,11 @@ rules:
     resources:
       - cninodes
     verbs: ["get", "list", "patch"]
+  - apiGroups: ["networking.k8s.aws"]
```

Comment:
small nit: can we move these new entries to before `CNINode`? Then they will match up with the release order

Reply:
Updated